lib/cosmos/tools/cmd_tlm_server/gui/replay_tab.rb

```diff
@@ -231,6 +248,31 @@ def select_log_file
       case packet_log_dialog.exec
       when Qt::Dialog::Accepted
         CmdTlmServer.replay_backend.select_file(packet_log_dialog.filenames[0], packet_log_dialog.packet_log_reader)
+        @op.setTitle("Playback Control: Log File")
+        @move_start.setEnabled(true)
+        @step_back.setEnabled(true)
+        @reverse_play.setEnabled(true)
+        @stop.setEnabled(true)
+        @play.setEnabled(true)
+        @step_forward.setEnabled(true)
+        @move_end.setEnabled(true)
+      end
+    end
+
+    def select_stream
+      stream_packets_dialog = StreamPacketsDialog.new(@widget, 'Select Stream', true)
```

Comment:
We need the ability to push in an initial start and end time so the previous selection can be remembered.

Reply:
The dialog has setters / getters to set the initial times if desired.